data/map.txt

```diff
@@ -28137,6 +28166,16 @@ planet Greymoon
 	bribe 0.1
 	security 0.5
 
+planet "Guardian Array Sapphire"
+	attributes heliarch arach "coalition station" 
+	landscape land/station33
+	description `More commonly referred to by its nickname "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war with the Quarg, meant to serve as a regional dry dock that damaged combat ships could return to after some time in the front lines. With no Quarg attack in so long, though, it's shifted into being a simple refuelling hub for passing merchant ships, as its dozens of warship-sized hangars remain mostly unused.`
+	description `	In spite of the lack of combat activity the station was designed to support, the local Heliarchs run a tight schedule even for civilian visitors as they first land, ensuring ships are refuelled and tended to as quickly as possible.`
+	spaceport `Built to serve as a purely strategic military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships on their way to a quick break, or heading back ready to depart and continue their business.`
+	spaceport `	On scheduled days of each month, Heliarch fleets that were damaged in training drills come in to stress test the station's repair capabilities, leading to a sudden boom in the station's population, and a crowding of the facilities...`
```

Comment:
```suggestion
	spaceport `	On certain days every month, Heliarch ships "damaged" during combat exercises dock to stress test the station's turnover capabilities, breathing new life in the station's population, and extreme overcrowding of the facilities...`
```

Reply:
I've never been sent overseas for exercises (Not yet my time to do so,) but my understanding is this. Battalion level exercises, especially for logistics, are a massive undertaking, neverminding the fact that you have to settle food for Muslims, make sure every piece of equipment is indented properly, and the minor stuff that can easily get you extra duties.

But for the most part of the year, these bases aren't fully staffed, so when you have a battalion, or even a brigade doing exercises, things are hectic, and most likely, shit has already hit the fan.

 The requested edits for this one is intended to replicate this in writing, so if you don't think it fits, contact me. I may or may not be able to answer depending on how busy I am.

Reply:
I don't mind the edits, can commit these.